staging/src/k8s.io/api/README.md

```diff
@@ -1 +1,20 @@
-This repo is still in the experimental stage. Shortly it will contain the schema of the API that are served by the Kubernetes apiserver.
+# api
+
+Schema of the external API types that are served by the Kubernetes apiserver.
+
+## Purpose
+
+This repo is still in the experimental stage.
+
+Canonical location of the Kubernetes API definition. Most likely interaction with this repository is as a dependency of client-go.
```

Comment:
Do we need the line about client-go?

Reply:
Oh, I see other repos mentioned in https://github.com/kubernetes/apiserver#purpose 👍